server/lib/pbench/process_tb.py

```diff
@@ -96,6 +96,9 @@ def process_tb(self) -> Results:
             tbdir = tb.parent
             controller = tbdir.name
 
+            if "::" in controller:
+                satellite_prefix, controller = controller.split("::")
+
```

Comment:
`satellite_prefix` is unused. I take it the plan is to add it to the metadata eventually?

Reply:
yes, that is the plan.